adage/analyze/tests.py

```diff
@@ -907,3 +928,100 @@ def test_experiment_filter_in_sample(self):
 
         # Test non-get methods too.
         self.call_non_get_API(filter_uri, data=data)
+
+    def create_expressionvalue_data(self, num_genes):
+        """
+        Generate test data for ExpressionValueResource
+        """
+        if Organism.objects.exists():
+            organism = Organism.objects.first()
+        else:
+            organism = factory.create(Organism)
+
+        if MLModel.objects.exists():
+            ml_model = MLModel.objects.first()
+        else:
+            ml_model = factory.create(MLModel)
+
+        # Create genes:
+        for i in range(num_genes):
+            Gene.objects.create(entrezid=(i + 1),
+                                systematic_name="sys_name #" + str(i + 1),
+                                standard_name="sys_name #" + str(i + 1),
```

Comment:
Typo: should be "std_name". This part of code seems to be copied from `create_edges()` function. Line #169 of this updated file has the same typo (my bad), can you correct it too? Thanks.